ca/certificates.go

```diff
@@ -209,23 +199,56 @@ func (rca *RootCA) RequestAndSaveNewCertificates(ctx context.Context, paths Cert
 		return nil, err
 	}
 
-	// Ensure directory exists
-	err = os.MkdirAll(filepath.Dir(paths.Cert), 0755)
+	var kekUpdate *KEKData
+	for i := 0; i < 5; i++ {
+		kekUpdate, err = rca.getKEKUpdate(ctx, X509Cert, tlsKeyPair, r)
```

Comment:
I think there is a backward compatibility issue here. Adding a new manager running 1.13 to a cluster with managers running 1.12 will fail. We need to support mixed deployments to make incremental upgrades possible. We should check specifically for `grpc.Code(err) == codes.Unimplemented` and treat that as if no KEK is set.